src/components/login/LoginContainer.js

```diff
@@ -138,7 +138,7 @@ const CustomContainer = styled(Container)`
    }
 `
 
-const LoginContainer = ({ loader, children, appTitle }) => (
+const LoginContainer = ({ loader, children, appTitle, contractId }) => (
```

Comment:
what does `contractId` mean in this context? The idea is that you create master account through wallet and then use this master account to deploy whatever contracts from CLI. Developer shouldn't need to login through wallet for every created contract.

Reply:
contract id is needed for the regular access key flow. It will be null for login.